pages/common/tee.md

```diff
@@ -0,0 +1,11 @@
+#TEE
```

Comment:
We already have a page about `tee` under osx and linux, as windows does not currently support `tee` by default. Perhaps you could merge your examples with those pages?